### ISSUE TITLE
Pretrained model, implementation details and performance difference.

### ISSUE THREAD

**Issue by XinyuYee** [leolyliu/SuperGlue_SuperPoint_MegaDepth#3]
Thank you very much for your great work! 
Is there a pre-trained model available? 

Or, could you please provide the implementation details, such as batch_size, epoch,learning_rate, sinkhorn_iterations of SuperGlue, keypoint_threshold, nms_radius of SuperPoint ..?

And, have you tested the performance difference between the best model of this code and the pre-trained model provided by SuperGLue?

Finally, thank you for your time! Looking forward to your reply.

**Comment by leolyliu** [comment]
The method implemented here can only support small-scale training with batch_size=1 for MegaDepth dataset. The dataloader needs some changes to support training with batch_size>1.

After training for 116,000 iterations with batch_size=1, the model's evaluation result is listed here: AUC@5=26.22，AUC@10=43.83，AUC@20=60.87，Prec=84.89，MScore=12.99.

In the subsequent experimental design, we've increased the batch_size to 8 and changed the algorithm of finding keypoint matches in the dataloader. The evaluation result of this method for 460,000 iterations of training could reach: AUC@5=32.54，AUC@10=52.27，AUC@20=69.63，Prec=97.86，MScore=16.78.

**Comment by XinyuYee** [comment]
> Thank you very much for your reply! Are the results you provide the result of pose estimation on the PhotoTourism dataset (just like Table3 in the Superglue paper)? If it is, then it really works out well! 

May I have your email address or other contact information? I am currently trying to reproduce the original accuracy of SuperGLue. Maybe we can work it out. By the way,  are you Chinese?  If so, we can communicate in Chinese.

**Comment by XinyuYee** [comment]
> 

Thank you very much for your reply! Are the results you provide the result of pose estimation on the PhotoTourism dataset (just like Table3 in the Superglue paper)? If it is, then it really works out well!

May I have your email address or other contact information? I am currently trying to reproduce the original accuracy of SuperGLue. Maybe we can work it out. By the way, are you Chinese? If so, we can communicate in Chinese.

**Comment by leolyliu** [comment]
The training dataset is MegaDepth, and the evaluation dataset is YFCC.

I'm Chinese. Email: liuyun18@mails.tsinghua.edu.cn
